operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -461,7 +462,6 @@ protected void addWarningsToStatus(Status status, Set<Condition> unknownAndDepre
     public Optional<LabelSelector> selector() {
         return selector;
     }
-
```

Comment:
I think we want to keep this empty line between methods?

Reply:
Same here ... we normally always have ampty line between two methods. No reason to delete it here.